### DISCUSSION TITLE
NaOH(s) in Extended UNIQUAC databases

### DISCUSSION THREAD

**Discussion by volpatto** [reaktoro/reaktoro#399] [Ideas]
Hi, @allanleal and community!

I have been working with multiples databases, and sometimes we need to convert one in another. Most of the time, this is fine, but I found a corner case that I would like to bring to discussion. The solid NaOH species in [ExtendedUNIQUAC.v2023](https://github.com/reaktoro/reaktoro/blob/main/embedded/databases/reaktoro/ExtendedUNIQUAC.v2023.yaml) has the `NaOH` name, which is fine if you stick to this DB alone. However, when converting to PHREEQC-like databases (for instance, the [Thermoddem DB](https://github.com/reaktoro/reaktoro/blob/main/embedded/databases/phreeqc/thermoddem-v1.10.dat#L4986)), `NaOH` corresponds to an aqueous species.

Here is the problem: I have been working with synonyms. So I define equivalent names. Considering SUPCRT07 and Thermoddem, `NaOH(aq)` and `NaOH` are synonyms, but `NaOH` is also a synonym for solid NaOH in ExtendedUNIQUAC.v2023. Unfortunately, I need to define a global synonym list with all the databases that I'm using, so the synonym list would always have E-UNIQUAC, SUPCRT and PHREEQC-like databases.

Given my explanation, I would like to ask you if we could change `NaOH` to `NaOH(s)` in ExtendedUNIQUAC.v2023 only, Allan. What do you think? Thanks a ton.

**Comment by allanleal** [comment]
Hi Diego. I think this is a reasonable request. To ensure backward compatibility, we could have a new ExtendedUNIQUAC.v2024 file that is exactly like ExtendedUNIQUAC.v2023 except for the names of solid species currently set to chemical formulas. Would that work for you?

**Reply by volpatto** [reply]
> Hi Diego. I think this is a reasonable request. To ensure backward compatibility, we could have a new ExtendedUNIQUAC.v2024 file that is exactly like ExtendedUNIQUAC.v2023 except for the names of solid species currently set to chemical formulas. Would that work for you?

Yeah, it would fit perfectly, if you agree. Thanks! 

Would you like to open a PR with this suggestion or should I take the lead?

**Reply by allanleal** [reply]
I had already created a PR, a few hours ago. Just merged. #400 

Thanks for the suggestion Diego.

**Reply by volpatto** [reply]
Thank you, @allanleal!
